src/main/resources/static/main.css

```diff
@@ -0,0 +1,20 @@
+.content {
+    width: 70%;
+    margin: auto;
+    padding-top: 4em;
+}
+
+table, th, td {
+    border: 1px solid black;
+    border-collapse: collapse;
+}
+
+th, td {
+    padding: 10px;
+}
+
+th {
+    text-align: left;
+    background-color: #D6EEEE;
```

Comment:
I don't know if we want to keep this color, or even the style overall, but maybe that will be the last issue to consider.

Reply:
Yes, I suggest we wait until later with the style, unless someone is excited and wants to work with the visuals :)